dev_guide/managing_images.adoc

```diff
@@ -116,6 +116,36 @@ endif::[]
 that prevents the registry client from pulling from such a tag.
 ====
 
+[[tag-naming]]
+=== Tag Naming
+
+Images evolve over time and tag reflects it. It always points to the latest
+image built. If there is too much information embedded in an tag name (e.g.
+`v2.0.1-may-2016`), the tag will probably point to just one revision of an
+image and will never be updated. Using default image pruning options, such an
+image will never be garbage collected. Instead, if the tag is named `v2.0`,
```

Comment:
s/garbage collected/removed

It may confuse users, garbage collection will be automatic mechanism, sometime in the future, pruning will always be manual, I think.